src/components/collapsible/Collapsible.js

```diff
@@ -50,11 +58,26 @@ class Collapsible extends React.Component<Props, State> {
 
     constructor(props: Props) {
         super(props);
+        this.transitionElement = React.createRef();
         this.state = {
             isOpen: props.isOpen,
+            height: '0',
```

Comment:
It seems like this should be conditional based on `props.isOpen`.

Reply:
At this point `this.transitionElement.current` would be null so you can't do a conditional. We can set to `auto` by default then use `componentDidMount` to update from there. There won't be any animation because you can't animate from auto.